code/modules/events/hostile_infestation.dm

```diff
@@ -0,0 +1,125 @@
+//Hostile Infestation: A variant of the infestation event that spawns small numbers of hostile mobs.
+//Currently only fires if there are 2 or more security officers
+
+#define LOC_KITCHEN 0
+#define LOC_ATMOS 1
+#define LOC_INCIN 2
+#define LOC_CHAPEL 3
+#define LOC_LIBRARY 4
+#define LOC_HYDRO 5
+#define LOC_VAULT 6
+#define LOC_TECH 7
+
+#define MONSTER_BEAR    0
+#define MONSTER_CREATURE 1
+#define MONSTER_XENO 2
+#define MONSTER_HIVEBOT  3
+#define MONSTER_ZOMBIE    4
+#define MONSTER_SKRITE  5
+#define MONSTER_SQUEEN  6
+
+/datum/event/hostile_infestation
+	announceWhen = 2  //Should notify crew much quicker than normal infestation event because of the danger posed by these mobs
+	endWhen = 10
+	var/localestring
+	var/monsterstring
+
+/datum/event/hostile_infestation/start()
+	to_chat(world, sound('sound/effects/bumpinthenight.ogg'))
```

Comment:
Are you sure a to_chat(world) is what you want? This will be heard by every single player connected to the server, even an unconscious deaf guy and a new player that hasn't joined yet.